lib/digest/crc15.rb

```diff
@@ -1,5 +1,15 @@
 require 'digest/crc'
 
+class Digest::CRC15 < Digest::CRC; end
+
+if RUBY_ENGINE == 'ruby'
+  begin;
+    require 'digest/crc15/crc15_ext'
+    return
```

Comment:
By returning early, it prevents the rest of the code from being defined. The C extensions only exist to override the existing `#update` method. The classes define other necessary methods, such as `self.pack` or `#finish`.

Reply:
:okay: Will try to get specs running locally and fix tomorrow or so.